src/Broadway/EventStore/Exception/DuplicatePlayheadException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Broadway\EventStore\Exception;
+
+use Broadway\EventStore\EventStoreException;
+use Exception;
+
+class DuplicatePlayheadException extends EventStoreException
+{
+    public static function create(Exception $exception)
```

Comment:
Yeah I had a really hard time here, the codebase isn't really consistent. However this was based on the DBALEventStoreException api.

I'll be happy to change it, when we agree upon format.

Thanks for the review @unkind